pontoon/base/views.py

```diff
@@ -104,6 +105,17 @@ def locale(request, locale, template='locale.html'):
     return render(request, template, data)
 
 
+def locales(request, template='locales.html'):
+    """Localization teams."""
+    log.debug("Localization teams.")
```

Comment:
Why do you log at the beginning of every view? If it's about determining which view function responded to a request, that seems like a job for a middleware.